<launch>

    <!-- Lanzar turtlesim -->
    <node
        pkg="turtlesim"
        exec="turtlesim_node"
        name="turtlesim"
        output="screen" />

    <!-- Nodo explorer con parÃ¡metros -->
    <node
        pkg="turtle_tracker"
        exec="explorer_node"
        name="explorer_node"
        output="screen">

        <param name="explorer_x" value="2.0"/>
        <param name="explorer_y" value="2.0"/>

    </node>

    <!-- Cliente del servicio turtle_info -->
    <node
        pkg="turtle_tracker"
        exec="client_node"
        name="info_client"
        output="screen" />

</launch>
